ui/src/app/pages/access/access-renewal-page.tsx

```diff
@@ -188,6 +262,12 @@ export const AccessRenewalPage = fp.flow(
         to maintain access to <AoU/> data. Renewal of access will occur on a rolling basis annually (i.e. for each user, access
         renewal will be due 365 days after the date of authorization to access <AoU/> data.
       </div>
+      {!anyModuleIsExpiring && <div style={{...renewalStyle.completionBox, gridColumnStart: 2}}>
```

Comment:
Something didn't work here - I didn't see this message after completing the appropriate steps.

Reply:
I should have mentioned: the backend changes need to be in place to fully test. I pulled in your changes when testing locally.
This could have happened if you saved the profile rather than clicked "Looks Good" as that change is not currently merged with the PR. I will merge with dev to bring this up to date.

Reply:
thx

Reply:
Discussed - it is not currently working if any are bypassed.  We agreed to defer a fix to the "polish" PR